internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -112,8 +111,24 @@ func TestBasicAndOLM(t *testing.T) {
 				t.Errorf("%s result State %v expected", result.Name, scapiv1alpha2.PassState)
 				return
 			}
-
 		})
+	}
+}
+func TestOLMBundle(t *testing.T) {
+	cases := []struct {
+		bundlePath string
+		state      scapiv1alpha2.State
+	}{
+		{"../testdata/bundle", scapiv1alpha2.PassState},
```

Comment:
Can you add a `FailState` test too?

Reply:
It doesn't look like we had the tests in the existing scorecard implementation, and since the scope of this PR is just updating the existing scorecard tests to the scorecard2 format, this may be better suited as a general follow-up PR to add more content to the scorecard2 testdata to more thoroughly exercise our failure scenarios. Likely all of the olm tests will be able to use the same failing testdata, so it will also be easier to add for all the tests once the actual tests have merged, as I think there are 4-5 open PRs adding various OLM/bundle tests.

Reply:
fyi, https://issues.redhat.com/browse/OSDK-1167